Monika After Story/game/script-greetings.rpy

```diff
@@ -2423,7 +2423,7 @@ label greeting_hairdown:
     # have monika's hair down
     $ monika_chr.change_hair(mas_hair_down, by_user=False)
 
-    call spaceroom(dissolve_masks=True, scene_change=True)
+    call spaceroom(dissolve_all=True, scene_change=True)
```

Comment:
anytime we call spaceroom now, we should be forcing an expression unless we want it to use idle-based expressions.  In this case, 1eua seems approprite.
```suggestion
    call spaceroom(dissolve_all=True, scene_change=True, force_exp="monika 1eua")
```